server/internal/infrastructure/db/sqlite/sqlc/query.sql

```diff
@@ -131,7 +131,7 @@ VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) ON CONFLICT(txid, vout) DO UPDATE SE
     swept = EXCLUDED.swept,
     expire_at = EXCLUDED.expire_at,
     redeem_tx = EXCLUDED.redeem_tx,
-    pending_change = EXCLUDED.pending_change;
+    pending = EXCLUDED.pending;
```

Comment:
do we really need this field ? pending isn't `redeem_tx != nil` ?

Reply:
yes, changes of async payments are marked as pending false